docker-compose.yml

```diff
@@ -22,6 +22,9 @@ services:
       - $PWD:/app
       # mount a dummy file to google application credentials, if not provided
       - ${GOOGLE_APPLICATION_CREDENTIALS:-./.add_credentials.md}:/tmp/.credentials
+      - $PWD/devdags:/app/devdags
```

Comment:
I'm not sure how ergonomic this will actually be. If I want to modify an existing workflow, I would have to eventually include the appropriate files and init variables. This is probably useful as standalone of the bigquery-etl workflow, though.

Reply:
I'm not sure what you mean. If modifying an existing dag, you would need to move it to the devdags folder, what other files and variables are you referring to? IIRC the current workflow only injects dummy variables, so you would need to import the real variables when testing anyways.  In terms of files, I mount the operators/utils folders into the devdags folder as well.

Reply:
It's also nice to only have to deal with the dag you're working on, and not load the others + vars/conns

Reply:
Example: if I want to test `dags/mozaggregator_prerelease.py`, I would have to copy it over to the devdags directory to test, then copy it back over. You're right about the variables because I do already have import scripts handy.

Reply:
Yeah this discussion is why I added you and fbertsch for review. 

IMO longer startup times, less logs, and little UI quirks (e.g. when toggling a dag on, and quickly refreshing, sometimes the wrong dag seems to be toggled on) would go away when only dealing with a single dag. 

The trade off is having to move dags before merging. But at least every new dag wouldn't require devs to add dummy vars/conns.

I did some testing and with prebuilt docker images, the startup time of local airflow without dags/conns/vars is ~10 seconds versus 3 minutes

Reply:
I don't have any real opinion here (as I won't be a consumer of this particular tooling), but 10s seems a lot nicer than 3m. Ergonomically though copying things around doesn't seem ideal.

Reply:
Testing on this dev cluster is going to be a pretty high-touch piece anyways, so copying over is not really a big deal. I think we should be good to go with this improvement as long as we properly document this workflow in the README: https://github.com/mozilla/telemetry-airflow#testing-gke-jobs-including-bigquery-etl-changes

Reply:
i've removed this piece and kept this pr involving gke only. If startup time is a goal in the future, we can revist this